study.md

```diff
@@ -56,13 +56,13 @@ If a class in Ruby gets modified while there are existing instances of it, then
 will those instances also have those modifications?
 
 ```md
-<!-- your answer here -->
+yes because classes contain all the instances of that class.
 ```
 
 ## Iteration
 
 Write an example of iteration in Ruby.
 
 ```ruby
-# your answer here
+[3, 45, 13, 94, 33].sort!
```

Comment:
We were looking for something along the lines of `[3, 45, 13, 94, 33].each { |n| puts n }`.